hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -165,7 +165,7 @@ public byte[] encodeTx() {
                     gasPrice,
                     Integers.toBytes(gasLimit),
                     to,
-                    Integers.toBytesUnsigned(value),
```

Comment:
There was a new bug when you pass a really big number(20 billion for example), we were wrapping it in **Integer** and then when we call `populateEthTxData` we were trying to unwrap it as **BigInteger**. To make it consistent I just used the value variable(which is **BigInteger**) to be consistent with `populateEthTxData`.

I wrote a new unit test to verify this and now the test-clients tests are passing.